packages/engine/README.md

```diff
@@ -109,53 +132,71 @@ Depending on your needs, different dependencies are required. Building this proj
+
+- `apt-get install build-essentials` - Includes the GCC/g++ compilers, libraries, and some other utilities
+- `apt-get install pkg-config` - A helper tool used when compiling applications and libraries
+- `apt-get install libssl-dev` - A development package of OpenSSL
+- `apt-get install python3-dev` - A collection of developer utilities for Python such as header files (e.g. needed when seeing `fatal error: Python.h: No such file or directory`)
 
 ### Project Setup / Building
-* Run `cargo build`
-* If you want to use a Python runner, also run `./src/worker/runner/python/setup.sh` and follow the instructions from the help
+
+- Run `cargo build`
+- If you want to use a Python runner, also run `./src/worker/runner/python/setup.sh` and follow the instructions from the help
 
 ### Running for development
+
 > **WIP** - This section is a work-in-progress. However, slightly more detailed documentation of the CLI is provided below in [CLI Arguments and Options](#cli-arguments-and-options).
 
 The CLI binary handles parsing a HASH project, and the lifetime of the engine for an experiment. To use it requires a HASH project to be accessible on the local disk. Follow instructions in the [Run a simulation](#run-a-simulation) section to learn how to download and create one.
 
 Then, run the CLI using:
-* `cargo run --bin cli -- <CLI ARGS>`
+
+```shell
+cargo run --bin cli -- <CLI ARGS>
+```
 
 Where CLI args are described below in the [Usage](#usage) section, an example of a run command during development would be:
-* `cargo run --bin cli -- <CLI ARGS> -p  "<PATH TO HASH PROJECT DIR>" single-run --num-steps <NUM-STEPS>`
+
+```shell
+cargo run --bin cli -- <CLI ARGS> -p "<PATH TO HASH PROJECT DIR>" single-run --num-steps <NUM-STEPS>
+```
 
 ## Quick Start Guide
 
 This guide will walk you through downloading a [demo simulation], running it, and then finding and verifying its output.
 
 In order to run the demo:
 
-0. Build the engine as [described above](#project-setup--building)
-1. Open the [demo simulation] and optionally read the overview
-2. Press `Open` at the upper right to view the simulation in [hCore].
-3. Download it by pressing `File -> Export Project`
-4. Unzip it either with your file browser or by e.g. `unzip ageing-agents.zip -d path/to/ageing-agents`
-5. Run the simulation from the _packages/engine_ directory and pass the path to the downloaded project as a parameter:
+1.  Build the engine as [described above](#project-setup--building)
+1.  Open the [demo simulation] and optionally read the overview
+1.  Press `Open` at the upper right to view the simulation in [hCore].
+1.  Download it by pressing `File -> Export Project`
+1.  Unzip it either with your file browser or by e.g. `unzip ageing-agents.zip -d path/to/ageing-agents`
+1.  Run the simulation from the _packages/engine_ directory and pass the path to the downloaded project as a parameter:
```

Comment:
I don't think this change is correct.

Reply:
I mean, I'm happy to use 1/1/1/1 (and it is my preference). But "0" is not "1", and "1" is not a sub-bullet of the first.

Reply:
Interesting observation, thanks @nonparibus! Turns out that both GitHub and VSCode support zero-based lists. I did not know that and thought they would be rendered rendered as `1. 2. 3.` anyway.

<img width="661" alt="Screenshot 2022-01-11 at 16 05 36" src="https://user-images.githubusercontent.com/608862/148979398-b51bd6af-1744-4f3b-8164-123ca74f03c9.png">

Spec: https://spec.commonmark.org/0.30/#example-267

I reverted back to `0. 1. 2.` in this particular instance by adding `<!-- markdownlint-disable ol-prefix -->` before the list. TBH I’m not sure we need `0. 1. 2.` so happy to revert this change.